src/components/alert/40917.vue

```diff
@@ -0,0 +1,46 @@
+<!--
+Copyright 2024 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div>
+    <span>{{ $tc('message', duplicateProperties.length) }}</span>
+    <ul>
+      <li v-for="p of duplicateProperties" :key="p.provided">
+        {{ $t('duplicateProperty', p) }}
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script setup>
+defineOptions({
+  name: 'Alert40917'
```

Comment:
I tried naming this `Alert409_17`, but the linter complained whenever I imported it:

```
Identifier 'Alert409_17' is not in camel case
```